lib/services/ios-project-service.ts

```diff
@@ -510,7 +510,15 @@ export class IOSProjectService extends projectServiceBaseLib.PlatformProjectServ
 
 			this.$logger.info("Installing pods...");
 			let podTool = this.$config.USE_POD_SANDBOX ? "sandbox-pod" : "pod";
-			return this.$childProcess.spawnFromEvent(podTool,  ["install"], "close", { cwd: this.platformData.projectRoot, stdio: 'inherit' }).wait();
+			let childProcess = this.$childProcess.spawnFromEvent(podTool,  ["install"], "close", { cwd: this.platformData.projectRoot, stdio: ['pipe', process.stdout, 'pipe'] }).wait();
+			if (childProcess.stderr) {
+				let errors = childProcess.stderr.replace(/(\u001b\[(?:\d*;){0,5}\d*m)[\s\S]+?(\u001b\[(?:\d*;){0,5}\d*m)/mg, "").trim();
```

Comment:
please use updated regex from @Mitko-Kerezov 
Also let's print the warnings with `$logger.warn` instead of just removing them.